index.html

```diff
@@ -493,6 +487,18 @@ <h1>SAVE Data (file0)</h1>
             </table>
           </div>
         </fieldset>
+        <fieldset class="col-span-3">
+          <legend>Advanced</legend>
+          <fieldset class="grid-cols-3" style="top: 3px; position: sticky;">
+            <div class="col-span-2">
+              <p>PLEASE only proceed if you know what you are doing!</p>
+              <p>Each flag has a mouseover explanation.</p>
+              <p><a href="https://pcy.ulyssis.be/undertale/flags" target="_blank">More</a> <a href="https://docs.google.com/document/d/1d5dt7A_jkTBhduVEGbWGtDbLEYMXCxSLUvqIqeXtYKw/" target="_blank">info</a></p>
```

Comment:
Thoughts on having this extracted into the help page alongside an explanation of the interface?

Reply:
I'm sure there's more useful information to add to the help page on that, but my thoughts are "that's a good idea" and "done".